Video/VideoIndexer/VideoIndexer.csproj

```
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
```

```suggestion
    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```
There's a mismatch between the csproj file and arm template causing the deployment to fail

---

oops that's embarrassing! Last change I made after I deployed it 😊

---

Fixed